askama_derive/src/generator.rs

```diff
@@ -73,9 +90,73 @@ fn build_template(ast: &syn::DeriveInput) -> Result<String, CompileError> {
     if input.print == Print::Code || input.print == Print::All {
         eprintln!("{code}");
     }
+
+    #[cfg(feature = "cache")]
+    cache_template(&digest, &code).unwrap();
+
     Ok(code)
 }
 
+#[cfg(feature = "cache")]
+fn find_cached_template(digest: &str) -> Option<String> {
+    let root = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
```

Comment:
Use `CARGO_TARGET_DIR` instead.

Reply:
`CARGO_TARGET_DIR` isn't intended to be used this way unfortunately. And on Windows, this isn't set during compilation. I think instead, we should re-export `OUT_DIR` in a build script.

Reply:
I just went with `OUT_DIR` since I can guarantee that it's set.